common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,49 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 createVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist createTwist(
+  const geometry_msgs::msg::Vector3 & velocity, geometry_msgs::msg::Vector3 & angular)
+{
+  geometry_msgs::msg::Twist twist;
+  twist.linear = velocity;
+  twist.angular = angular;
+  return twist;
+}
+
+inline double calcTwistNorm(const geometry_msgs::msg::Vector3 & v)
+{
+  return std::hypot(v.x, v.y, v.z);
+}
+
+inline double calcTwistNorm(const geometry_msgs::msg::Twist & twist)
+{
+  return calcTwistNorm(twist.linear);
+}
```

Comment:
I think this wrapper isn't necessary. Let's use `calcNorm(twist.linear)` in the application code.
Also, for `Vector3`, the word `Twist` is unnecessary.

```suggestion
inline double calcNorm(const geometry_msgs::msg::Vector3 & v)
{
  return std::hypot(v.x, v.y, v.z);
}
```

Reply:
fixed at [bfcd5be](https://github.com/autowarefoundation/autoware.universe/pull/2187/commits/bfcd5be2df96b3a97c77e39947cd0c6c84714b3a)